documentation/todo-app/src/App.js

```diff
@@ -1,18 +1,14 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import TodoList from './components/todoList';
```

Comment:
It's better to start the name of the component with Capital letter `import TodoList from './components/TodoList';`